apps/design-system-docs/docs/components/ds/feedback/index.tsx

```diff
@@ -0,0 +1,5 @@
+export * as badge from './badge.mdx';
+export * as notice from './notice.mdx';
+export * as spinner from './spinner.mdx';
+export * as snackbar from './snackbar.mdx';
+export * as progressBar from './progress-bar.mdx';
```

Comment:
This index file has nothing to do with the actual rendering of the MDX pages on the site. It's just so we can access the metadata of the MDX files programmatically.